public/app/percona/shared/components/PerconaBootstrapper/PerconaUpdateVersion/PerconaUpdateVersion.tsx

```diff
@@ -62,7 +57,7 @@ const PerconaUpdateVersion: FC = () => {
         className={styles.updateVersionModal}
       >
         <div data-testid="one-update-modal">
-          <h5 className={styles.version}>{changeLogs?.updates[0]?.version || ''}</h5>
+          <h5 className={styles.version}>{latest?.version || ''}</h5>
           <p className={styles.releaseNotesText}>
             <a target="_blank" rel="noopener noreferrer" href={changeLogs?.updates[0]?.releaseNotesUrl || ''}>
```

Comment:
I think you can `latest` here also. (if it contains the releaseNotesUrl, not 100% sure)

Reply:
fixed